course/layouts/pages/section.html

```diff
@@ -0,0 +1,3 @@
+{{ define "main" }}
+{{ partial "course_content.html" . }}
+{{ end }}
```

Comment:
Currently, this code
```
{{ define "main" }}
{{ partial "course_content.html" . }}
{{ end }}
```
is being repeated in 3 files: 
1- course/layouts/pages/**single**.html
2- course/layouts/pages/**section**.html
3- course/layouts/pages/**instructor_insights**.html

what if we write this common code in a common file and import it from there? So we can eliminate this code duplication? 

I'm also wondering, what could be the reason for creating an additional/separate `instructor insights` layout, if it is to manage lists, then I guess it can be done with `section.html` as well.

@gumaerc Can you also share your thoughts regarding this?

Reply:
> I'm also wondering, what could be the reason for creating an additional/separate instructor insights layout, if it is to manage lists, then I guess it can be done with section.html as well.

This is a good question. In general, we don't like special cases like this. However, the legacy content for instructor insights _is_ structured differently than the rest of the site, _and_ the OCW team is interested in having it look different. We are still discussing how different it might be, but in the short term, I think it's ok to continue having a unique theme for this page and it's children. 

I'm interested to hear @gumaerc's thoughts too.

Reply:
@gumaerc Thank you for this detailed explanation although I still have one question.
Should I merge this PR?